docs/source/examples.rst

```diff
@@ -529,5 +529,28 @@ request.
               {{ check_conditions }}
 
 
+🐍 Updating PR for linear history
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As GitHub recently introduced linear history in PR setting it become very handy to use an automatic tool
+ to keep your PRs up-to-date. On the other hand you do not want to overkill your CI with building all PRs
+ always especially when there is still work in progress, so you can limit this action just to ready labeled PRs.
+
+.. code-block:: yaml
+
+    pull_request_rules:
+      - name: automatic update for PR marked as “Ready-to-Go“
+        conditions:
+          - -conflict # skip PRs with conflicts
+          - -draft # filter-out GH draft PRs
+          - label="Ready-to-Go"
+        actions:
+          update: {}
+
+.. note::
+
+    Mergify is event driven bot so even you add this cool rule, it wound be executed till each PR is updated
+     or the bot is pinged in a comment as `@Mergifyio refresh`.
+
```

Comment:
We should write that somewhere else in a more "general" place.
```suggestion

```

Reply:
so at the beginning or where bc till I knew it caused me lost of troubles to understands what is happening

Reply:
I'll probably add this to https://docs.mergify.io/getting-started.html